### PR TITLE
Fix more info in package error flash

### PR DESCRIPTION
**Pull request by Ana06**
The Javascript for the more info was missing and  `.hidden` was been dropped in Bootrstrap 4: https://getbootstrap.com/docs/4.0/migration. Also, style link in alert using Bootstrap class.  :bowtie: 

Fixes https://github.com/openSUSE/open-build-service/issues/6072

image

image

### CODE REVIEW

File: src/api/app/assets/javascripts/webui2/packages.js [hunk 1]
```diff
@@ -34,3 +34,11 @@ $(function ($) {
     details.attr('open', null);
```
Comment (Ana06) [inline src/api/app/assets/javascripts/webui2/packages.js]: I wonder why this file is called `packages.js` (in plural) and not `package.js`? :thinking: Any clue @openSUSE/open-build-service ? :thinking:

Comment (vpereira) [reply src/api/app/assets/javascripts/webui2/packages.js]: @ChrisBr could answer that, but why we have ```package``` and ```packages```? Maybe in the past it used to be all ```packages``` ?

Comment (hennevogel) [reply src/api/app/assets/javascripts/webui2/packages.js]: Why do you think this matters?

Comment (Ana06) [reply src/api/app/assets/javascripts/webui2/packages.js]: @hennevogel 

> Why do you think this matters?

consistency? I just didn't know where to put it until I found out that this was for package related things.. there is also a `package-view_file.js` (in singular). It confuses me... :roll_eyes:

File: src/api/app/assets/javascripts/webui2/packages.js [hunk 2]
```diff
@@ -34,3 +34,11 @@ $(function ($) {
...
   });
 });
+
+$(document).ready(function() {
+  $('.btn-more').click(function() {
+    var moreInfo = $('.more_info');
+    moreInfo.toggleClass('d-none');
+    $(this).text(moreInfo.hasClass('d-none') ? 'more info'  : 'less info');
+  });
```
Comment (eduardoj) [inline src/api/app/assets/javascripts/webui2/packages.js]: Nitpick: two spaces after 'more info'. Should be one.

File: src/api/app/assets/javascripts/webui2/packages.js (hunk 1 above)
Comment (hennevogel) [reply src/api/app/assets/javascripts/webui2/packages.js]: None of this is related to anything. Its one huge javascript file that get's included on every page. However you structure it internally is up to you :smile:

Comment (Ana06) [reply src/api/app/assets/javascripts/webui2/packages.js]: I think we need some criteria for JavaScript file names, before this is a mess... :bulb:

**Comment by Ana06** [comment]
**checkout_code** is broken... :angry:

**Comment by Ana06** [comment]
:dancer: :tada:

**Pull request merged**